linkerd/admin/src/main/resources/io/buoyant/linkerd/admin/js/router_clients.js

```diff
@@ -0,0 +1,105 @@
+var RouterClient = (function() {
+  var template;
+
+  function getMetricDefinition(routerName, clientName) {
+    return [
+      {
+        description: "Requests",
+        metricSuffix: "requests",
+        query: Query.clientQuery().withRouter(routerName).withClient(clientName).withMetric("requests").build()
```

Comment:
tioli: I might cut some of the duplication by defining a baseQuery() function, i.e.

```js
function baseQuery(routerName, clientName) {
  return Query.clientQuery().withRouter(routerName).withClient(clientName);
}
```
and then you could do `baseQuery().withMetric("requests").build()` here and below. Actually, I would have expected to be able to make baseQuery a var since often I see methods named `withFoo` make copies rather than mutating. In a follow-up branch, I might suggest either changing those methods to be called `setFoo` or to have them copy the original query, but that's certainly not worth doing here.